ChustaSoft.Common.AspNet/Extensions/ControllerBaseExtensions.cs

```diff
@@ -1,9 +1,8 @@
-﻿using Microsoft.AspNetCore.Mvc;
-using System;
+﻿using System;
 using System.IdentityModel.Tokens.Jwt;
 using System.Linq;
 
-namespace ChustaSoft.Common
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
Will create naming conflicts with https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc?view=aspnetcore-6.0

Reply:
Should not be the case, only proving a couple of new methods for ControllerBase under the same.

The purpose of this is to extend the methods provided by Microsoft namespace without requiring add a new external namespace in the client app code.